core/input/input_map.cpp

```diff
@@ -319,7 +319,11 @@ void InputMap::load_default() {
 	key->set_keycode(KEY_END);
 	action_add_event("ui_end", key);
 
-	//set("display/window/handheld/orientation", "landscape");
+	add_action("toggle_fullscreen");
```

Comment:
Is `ui_` prefix needed? All system actions have it.

Reply:
This reminds me, it would be nice to have a checkbox to hide built-in actions (I think there's a proposal somewhere but idk). One easy way to do this would be to hide everything starting with `ui_`, but if `toggle_fullscreen` doesn't then that would have to be hard-coded too.

Reply:
I'm not sure if `ui_toggle_fullscreen` would really make sense, so I called it just `toggle_fullscreen`.

Maybe we should revisit the name of built-in actions to distinguish them with something else than `ui_` (maybe a leading underscore?).

Reply:
I'd use the `ui_` prefix for now to be consistent.